h/static/scripts/annotation-ui.js

```diff
@@ -52,7 +52,10 @@ function initialState(settings) {
 
     filterQuery: null,
 
-    sortMode: 'Location',
+    // Key by which annotations are currently sorted.
+    sortKey: 'Location',
+    // Keys by which annotations can be sorted.
+    sortKeysAvailable: ['Newest', 'Oldest', 'Location'],
```

Comment:
As the code is right now that makes sense. One reason that it can be useful to use objects to group fields as the state gets more complex is that makes it easier to split up the main `(state, action) => nextState` update function into smaller ones which each take a specific part of the existing state as input and return the new version of that piece as a result.